Orbitersdk/samples/ProjectApollo/src_launch/RTCC_Mission_C_PRIME.cpp

```diff
@@ -457,9 +457,11 @@ bool RTCC::CalculationMTP_C_PRIME(int fcn, LPVOID &pad, char * upString, char *
 		{
 			sprintf(upMessage, "%s has been scrubbed.", manname);
 
-			//Always save table 2 data for MCC-1
+			//Always save LOI time and table 2 data for MCC-1, even if the maneuver is scrubbed
 			if (fcn == 20)
```

Comment:
This block is now identical with the one in the else case. So it should be able to move this out of `if (scrubbed)` and simply do `if (fcn == 20 || fcn == 21)`